docs/src/operate/c8y/cloud-profiles.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Cloud Profiles
+tags: [Operate, Cumulocity, Cloud Profile]
+description: Connecting %%te%% to multiple Cumulocity tenants
+---
+
+import UserContext from '@site/src/components/UserContext';
+import UserContextForm from '@site/src/components/UserContextForm';
+
+:::tip
+#### User Context {#user-context}
+
+You can customize the documentation and commands shown on this page by providing
+relevant settings which will be reflected in the instructions. It makes it even
+easier to explore and use %%te%%.
+
+<UserContextForm settings="C8Y_PROFILE_NAME,C8Y_PROFILE_URL,C8Y_URL,DEVICE_ID" />
```

Comment:
@reubenmiller I'm wondering whether I actually want `C8Y_URL` and `DEVICE_ID` here. The device id certainly affects nothing people will copy and paste (just the `tedge connect` output, and I think the additional URL is just going to make things more confusing, since people have presumably followed a separate guide.

If we do want `C8Y_URL` here, I'm tempted to put `C8Y_PROFILE_URL` before it in the form since that's the most pertinent configuration value here.

Reply:
Though `C8Y_URL` is still used in various places (when showing the user the interaction between setting the default/primary c8y.url and the second cloud profile).

So we can leave it as-is for now, and look at adding some more context to the form fields (e.g. maybe a description of what the field actually means etc.)